ozaria/site/components/teacher-dashboard/BaseCurriculumGuide/components/ChapterInfo.vue

```diff
@@ -141,7 +153,11 @@ export default {
     campaigns.fetchByType('course', { data: { project: 'levels,levelsUpdated' } })
     campaigns.on('sync', () => {
       const campaign = campaigns.get(this.getCurrentCourse.campaignID)
-      this.levelsNameMap = campaign.getLevelNameMap()
+      if (campaign) {
+        this.levelsNameMap = campaign.getLevelNameMap()
+      } else {
+        // Campaign is not loaded yet, do we need to do anything here?
```

Comment:
why can campaign is not loaded when campaigns is on `sync` ?

Reply:
I'm not sure, I put this in because of an error with `campaign.getLevelNameMap()` where `campaign` was null.